docs/upgrading.md

```diff
@@ -5,6 +5,10 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.2.1.post1` to `v0.2.2`
+
+No backwards incompatible changes
```

Comment:
AFAIC!

Reply:
I think only in tests...and so yeah hopefully you're right